src/main/webapp/intrigue-api/graphql.js

```diff
@@ -7,7 +7,7 @@ import { mergeDeepOverwriteLists } from '../utils'
 const { BatchHttpLink } = require('apollo-link-batch-http')
 const { genSchema, toGraphqlName, fromGraphqlName } = require('./gen-schema')
 
-const ROOT = '/search/catalog/internal'
+const ROOT = 'https://localhost:8993/search/catalog/internal'
```

Comment:
why is this change needed?

Reply:
I kept getting `only absolute urls are supported` errors until I did that in my tests of the GraphQL queries in `keyword.tsx` Not sure why but I just reverted this line.